backends/rapidpro/msg.go

```diff
@@ -343,6 +345,7 @@ type DBMsg struct {
 	Text_         string                 `json:"text"          db:"text"`
 	Attachments_  pq.StringArray         `json:"attachments"   db:"attachments"`
 	ExternalID_   null.String            `json:"external_id"   db:"external_id"`
+	Metadata_     null.String            `json:"metadata"      db:"metadata"`
```

Comment:
It would have to be an interface map since this isn't a nice one level deep config. Decided it made more sense to expose things at a higher level instead. (IE QuickReply instead of metadata at the Msg level)